.azure/pipelines/devBuilds.yml

```diff
@@ -31,12 +31,12 @@ stages:
       steps:
       - script: git submodule init
       - script: git submodule update --recursive
-      - script: cd ./src/Components
```

Comment:
@mkArtakMSFT you may want to look at this file in particular

Reply:
@pranavkm can you please look into this and see whether this change is correct ? Thanks!

Reply:
Looks fine to me. I initially have totally forgotten that this is something I had created. Thanks @pranavkm for reminding this 👍 
Anyway, the change looks reasonable. We should discuss in our next meeting whether this is still useful. We tend to look at the official CI views rather than this any more, so maybe it's time to completely stop this.